_data/projects/infisical.yml

```
@@ -0,0 +1,16 @@
+name: Infisical
+desc: Infisical is an open-source, E2EE tool to sync environment variables across your team and infrastructure.
+site: https://infisical.com/
+tags:
+  - security
+  - secret-management
+  - environment-variables
+  - end-to-end-encryption
+  - cli
+  - web
+  - node.js
+  - go
+  - next.js
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Infisical/infisical/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

:art: we can shorten this link

```suggestion
  link: https://github.com/Infisical/infisical/labels/help%20wanted
```